src/miner.cpp

```diff
@@ -728,6 +731,16 @@ namespace pos {
 
         auto pblock = std::make_shared<CBlock>(pblocktemplate->block);
 
+        // Set search time if null or last block has changed
+        if (!nLastCoinStakeSearchTime || lastBlockSeen != tip->GetBlockHash()) {
+            if (Params().NetworkIDString() == CBaseChainParams::REGTEST) {
+                nLastCoinStakeSearchTime = GetAdjustedTime();
+            } else {
+                nLastCoinStakeSearchTime = tip->GetMedianTimePast() + 1;
```

Comment:
Plus 1 is for what purpose?

Reply:
Median time or less will fail with time-too-old, I'll add a comment.